google-workspace.sgnl.yaml

```diff
@@ -0,0 +1,277 @@
+      - name: orgUnitPath
+        externalId: orgUnitPath
+        type: String
+      - name: isMailboxSetup
+        externalId: isMailboxSetup
+        type: Bool
+      - name: isEnrolledIn2Sv
+        externalId: isEnrolledIn2Sv
+        type: Bool
+      - name: isEnforcedIn2Sv
+        externalId: isEnforcedIn2Sv
+        type: Bool
+      - name: includeInGlobalAddressList
+        externalId: includeInGlobalAddressList
+        type: Bool
+      - name: recoveryEmail
+        externalId: recoveryEmail
+        type: String
+  UserEmail:
+    parent: User
+    displayName: UserEmail
+    externalId: emails
+    description: Entity representing an email object for a Google Workspace User
+    pageSize: 100
+    pagesOrderedById: false
+    attributes:
+      - name: address
+        externalId: address
+        type: String
+        indexed: true
+        uniqueId: true
+      - name: type
+        externalId: type
+        type: String
+      - name: primary
+        externalId: primary
+        type: Bool
+  # https://developers.google.com/admin-sdk/directory/reference/rest/v1/groups#Group
+  Group:
+    displayName: Group
+    externalId: Group
+    description: Entity representing a Google Workspace Group
+    pageSize: 100
+    pagesOrderedById: false
+    attributes:
+      - name: email
+        externalId: email
+        type: String
+        indexed: true
+        uniqueId: true
+      - name: id
+        externalId: id
+        type: String
+        indexed: true
+        attributeAlias: groupId
+      - name: kind
+        externalId: kind
+        type: String
+      - name: etag
+        externalId: etag
+        type: String
+      - name: name
+        externalId: name
+        type: String
+      - name: directMembersCount
+        externalId: directMembersCount
+        type: String
+      - name: adminCreated
+        externalId: adminCreated
+        type: Bool
+      - name: nonEditableAliases
+        externalId: nonEditableAliases
+        type: String
+        list: true
+  # https://developers.google.com/admin-sdk/directory/reference/rest/v1/members#Member
+  # In Google Workspace, a member can be a user (GroupMember) or a group (SubGroup).
+  Member:
+    displayName: Member
+    externalId: Member
+    description: Entity representing a Google Workspace Member
+    pageSize: 100
+    pagesOrderedById: false
+    attributes:
+      - name: id
+        externalId: uniqueId
+        type: String
+        indexed: true
+        uniqueId: true
+      - name: memberId
+        externalId: id
+        type: String
+        indexed: true
+        attributeAlias: memberId
+      - name: groupId
+        externalId: groupId
+        type: String
+        indexed: true
+        attributeAlias: memberGroupId
+      - name: kind
+        externalId: kind
+        type: String
+      - name: etag
+        externalId: etag
+        type: String
+      - name: email
+        externalId: email
+        type: String
```

Comment:
What email is this referring to? The user or the group? If user, then let's update the `name` to `userEmail` that instead of ~~email~~.

Reply:
fixed